CHANGES.md

```diff
@@ -9,13 +9,8 @@ Unreleased
 - Add `dune build --dump-gc-stats FILE` argument to dump garbage collection
   stats to a named file. (#8072, @Alizter)
 
-3.9.1 (2023-07-06)
-------------------
-
-- Disable background operations and threaded console on MacOS and other Unixes
-  where we rely on fork. (#8100, #8121, fixes #8083, @rgrinberg, @emillon)
-
-- Initialize async IO thread lazily. (#8122, @emillon)
+- `enabled_if` now supports `arch_sixtyfour` variable (#8023, fixes #7997,
```

Comment:
Needs to be updated

Reply:
I've updated it now, not sure what happened.